src/org.hdfgroup.hdfview/hdf/view/dialog/UserOptionsHDFPage.java

```diff
@@ -191,6 +229,30 @@ protected void load() {
         checkIncOrder.setSelection(indexOrder.compareTo("H5_ITER_INC") == 0);
         checkDecOrder.setSelection(indexOrder.compareTo("H5_ITER_DEC") == 0);
         checkNativeOrder.setSelection(indexOrder.compareTo("H5_ITER_NATIVE") == 0);
+
+        pluginDir = ViewProperties.getPluginDir();
+        if (pluginDir == null)
+            pluginDir = rootDir;
```

Comment:
Should this be `rootDir + "/plugin"` to match the default in ViewProperties.java?

Reply:
Probably, it should just be a default if none are present, situation.